### PR TITLE
iOS: Install resources from app manifest

### PR DESCRIPTION
**Pull request by tido64**

### CODE REVIEW

**Comment by tido64** [comment]
@alloy: How do you feel about generating a temporary resources podspec?

**Comment by alloy** [comment]
It’s not the worst thing 🤔 I was wanting to see about generating an in-memory podspec, but this may actually be more API proof.

**Comment by tido64** [comment]
I looked into it yesterday. Worked back from [Core/dsl.rb](https://github.com/CocoaPods/Core/blob/master/lib/cocoapods-core/podfile/dsl.rb), got as far as the [analyzer](https://github.com/CocoaPods/CocoaPods/blob/e5485d55addf291a61f1eae9bfc2f1fe1fe95b18/lib/cocoapods/installer/analyzer.rb#L101), and gave up. I couldn't find a proper way to inject `PodTargets`. Not sure about this solution myself.

File: test_app.rb [hunk 1]
```diff
@@ -82,6 +108,9 @@ def use_test_app!(package_root)
     pod 'glog', :podspec => "#{react_native}/third-party-podspecs/glog.podspec"
     pod 'Folly', :podspec => "#{react_native}/third-party-podspecs/Folly.podspec"
 
+    resources_pod_path = resources_pod(package_root)
+    pod 'ReactTestApp-Resources', :path => resources_pod_path unless resources_pod_path.nil?
```
Comment (alloy) [inline test_app.rb]:
```ruby
if resources_pod_path = resources_pod(package_root)
  pod 'ReactTestApp-Resources', :path => resources_pod_path
end
```

**Comment by tido64** [comment]
@alloy: Should we go ahead with this for now and figure out a better implementation later? The underlying implementation should be invisible to consumers in any case.

**Pull request merged**